src/main/java/com/skankhunt220/service/CityService.java

```diff
@@ -0,0 +1,53 @@
+package com.skankhunt220.service;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import com.skankhunt220.entity.City;
+import com.skankhunt220.repository.city.CityRepository;
+
+@Service
+public class CityService {	
+	@Autowired
+	private CityRepository cityRepository;
+
+	public ResponseEntity<City> createCity(City city) {
+		try {
+			return ResponseEntity.status(HttpStatus.CREATED)
+					.body(cityRepository.save(new City(city.getName(), city.getZipCode())));
+		} catch (Exception e) {
+			return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
+		}
+	}
+	
+	public ResponseEntity<City> readCity(String cityId) {
+		Optional<City> cityData = cityRepository.findById(cityId);
+		if (cityData.isPresent()) {
```

Comment:
Don't split response on found not found, just return response, if response is null, then object is not found